packages/runtime/src/controls/rich-text/__tests__/dto.test.ts

```diff
@@ -1,60 +0,0 @@
-import { blockquoteFixture, listFixture, typographyFixture } from '../fixtures'
-import { richTextDAOToDTO, richTextDTOtoDAO, richTextDTOtoSelection } from '../dto'
-import { RichTextDTO } from '../dto-types'
-import { emptyBlockFixture, fixedEmptyBlockFixture } from '../fixtures/empty-blocks'
-import { v2DataFixture } from '../fixtures/v2-data-types'
-
-describe('GIVEN I am using RichText dto translation', () => {
-  test('WHEN translating simple data THEN nothing is lost', () => {
-    const value: RichTextDTO = {
-      document: {
-        object: 'document',
-        data: undefined,
-        nodes: [],
-      },
-      annotations: undefined,
-      data: undefined,
-      object: 'value',
-      selection: undefined,
-    }
-
-    expect(value).toStrictEqual(
-      richTextDAOToDTO(richTextDTOtoDAO(value), richTextDTOtoSelection(value)),
-    )
-  })
-  test('WHEN translating list data THEN nothing is lost', () => {
-    expect(listFixture).toStrictEqual(
-      richTextDAOToDTO(richTextDTOtoDAO(listFixture), richTextDTOtoSelection(listFixture)),
-    )
-  })
-  test('WHEN translating typography data THEN nothing is lost', () => {
-    expect(typographyFixture).toStrictEqual(
-      richTextDAOToDTO(
-        richTextDTOtoDAO(typographyFixture),
-        richTextDTOtoSelection(typographyFixture),
-      ),
-    )
-  })
```

Comment:
Do we have equivalent tests for these?

Reply:
We do: https://github.com/makeswift/makeswift/blob/aleksey/eng-5464-part2/packages/controls/src/controls/rich-text/dto/dto.test.ts